app/src/androidTest/res/raw/unfriendly_activity_report.json

```diff
@@ -12,7 +12,7 @@
         ]
     },
     {
```

Comment:
Hmm, shouldn't this need an update for the new item in the list or does it just not show up when loading the activity?

Reply:
This is related to [the comment](https://github.com/quittle/a11y-ally/pull/346#discussion_r912414674), I'll revert this change.